src/appshell/view/preferences/appearancepreferencesmodel.h

```diff
@@ -58,6 +58,8 @@ class AppearancePreferencesModel : public QObject, public async::Asyncable
     Q_PROPERTY(
         QString foregroundWallpaperPath READ foregroundWallpaperPath WRITE setForegroundWallpaperPath NOTIFY foregroundWallpaperPathChanged)
 
+    Q_PROPERTY(bool invertScoreColor READ invertScoreColor WRITE setInvertScoreColor NOTIFY invertScoreColorChanged)
```

Comment:
it is better to rename to scoreInversionEnabled

Reply:
Fixed